.circleci/config.yml

```diff
@@ -250,9 +250,7 @@ jobs:
       - run:
           name: Build Hermes for macOS as CocoaPods pod
           command: |
-            cd "$HERMES_WS_DIR"
-            hermes/utils/build/configure.py --distribute --cmake-flags="$(ruby -rcocoapods-core -e 'load %{hermes/hermes.podspec}; puts HermesHelper.cmake_configuration') -DCMAKE_INSTALL_PREFIX:PATH=../destroot"
-            cmake --build ./build_release --target hermes-runtime-darwin-cocoapods-release
+            exit 1
```

Comment:
Left as a todo. I need to replace `--cmake-flags="$(ruby -rcocoapods-core -e 'load %{hermes/hermes.podspec}; puts HermesHelper.cmake_configuration') -DCMAKE_INSTALL_PREFIX:PATH=../destroot"` with `./build-apple-framework.sh`.

It is trivial, but I want to take time to connect that with building iOS pieces as well.